### ISSUE TITLE
Compiling with xcodebuild?

### ISSUE THREAD

**Issue by pedro** [facebook/react-native#369]
Is anyone able to compile a react-native app with `xcodebuild`? I can compile regular Swift apps normally, but not React:

```
$ xcodebuild clean build -scheme MyApp
2015-03-27 14:10:33.203 xcodebuild[2284:52159] [MT] DVTAssertions: ASSERTION FAILURE in /SourceCache/IDEXcode3ProjectSupport/IDEXcode3ProjectSupport-6728/Xcode3Core/LegacyProjects/Frameworks/DevToolsCore/DevToolsCore/RuntimeSupport/SDKs/XCSDKPackage.m:202
Details:  Assertion failed: [basePath isAbsolutePath]
Object:   <XCSDKPackage>
Method:   +sdkPackageForNameOrPath:withBasePath:forceCreate:
Thread:   <NSThread: 0x7fde03c2e4c0>{number = 1, name = main}
Hints: None
Backtrace:
  0  0x000000010c95673a -[DVTAssertionHandler handleFailureInMethod:object:fileName:lineNumber:assertionSignature:messageFormat:arguments:] (in DVTFoundation)
  1  0x000000010c95618f _DVTAssertionHandler (in DVTFoundation)
  2  0x000000010c95647e _DVTAssertionFailureHandler (in DVTFoundation)
  3  0x000000010c9563e0 _DVTAssertionFailureHandler (in DVTFoundation)
  4  0x000000010e3a49f7 +[XCSDKPackage sdkPackageForNameOrPath:withBasePath:forceCreate:] (in DevToolsCore)
  5  0x000000010e261f2d -[PBXTarget createMacroExpansionScopeWithBuildParameters:] (in DevToolsCore)
  6  0x000000010e263807 -[PBXTarget cachedMacroExpansionScopeForBuildParameters:] (in DevToolsCore)
...
```

Not experienced with iOS dev so wasn't able to make much progress with this. fwiw trying to list build targets with `xcodebuild -list -project` yields the same error. Running Xcode 6.2, build 6C131e.

**Comment by corymsmith** [comment]
@pedro The fix for this is #344

**Comment by pedro** [comment]
sweet, works now! thanks
